scripts/set-linux-desktop-file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+respath=$1
+iconpng=$2
+iconsvg=$3
+desktop=$4
+mkdir -p $respath
+mkdir -p ~/.local/share/applications/
```

Comment:
Could you set these hardcoded paths (as an arg) to `${CMAKE_INSTALL_PREFIX}/...` as appropriate? I assume `${CMAKE_INSTALL_PREFIX}/share/` is correct, but please double check.

Reply:
In fact, this isn't required at all, since the `.desktop` file handles it. So I'll remove it.

Reply:
I _think_ the proper way is to try to conform to the XDG conventions and put it into $HOME/.local. So, good to go here.